tests/home-page-check

```diff
@@ -0,0 +1,9 @@
+import { test } from "@framework/test";
```

Comment:
@lipmaneth see @0xLienid 's comment about using the base `@playwright/test` framework instead of the proprietary guardianui `@framework/test`.